test/auth.test.js

```diff
@@ -63,29 +71,27 @@ describe('Test Suite for auth', () => {
         done();
     });
     
-    test('POST /signup with same email of existing user should respond with 400', async (done) => {
-        const res = await request(server).post('/signup').send(user);
+    test('POST /api/signup with same email of existing user should respond with 400', async (done) => {
+        const res = await request(server).post('/api/signup').send(user);
         // expect error since can only have one account per email
         expect(res.statusCode).toBe(400);
         done();
     });
 
-    // POST /login
-    test('POST /login with no data should respond with 400', async (done) => {
-        const res = await request(server).post('/login');
-	// expect error since missing data
+    // POST /api/login
+    test('POST /api/login with no data should respond with 400', async (done) => {
+        const res = await request(server).post('/api/login');
+        // expect error since missing data
         expect(res.statusCode).toBe(400);
```

Comment:
The indentation had to be corrected in `auth.test.js` and `auth.controller.js` as mentioned on slack, @ernewtoner. Also, VSCode defaults to 2 spaces, so just be sure that you're using 4 spaces, so we don't have to fix the formatting. 😬🙏

Reply:
I switched to vscode :+1:

Reply:
that's why prettier is awesome! then you don't have to think about formatting :)

Reply:
@ernewtoner, yep, I got that you're using VSCode. It defaults to 2 spaces, so just be sure you're using 4 spaces. You need to manually make that change or you can use a plugin/ext for prettier.

And oh yeah, I used prettier locally @mnicole. I just wanted to be sure we were all using the same basic formatting though, so we don't have to commit the same files repeatedly just for formatting changes!

Reply:
Ah yeah sorry was thinking back to our previous convo about VSCode--set it to 4 spaces.